pre_commit_hooks/detect_aws_credentials.py

```diff
@@ -84,7 +84,7 @@ def check_file_for_aws_keys(
             for key in keys:
                 # naively match the entire file, low chance of incorrect
                 # collision
-                if key in text_body:
+                if key and key in text_body:
```

Comment:
can you add a quick test for this to demonstrate the change -- definitely agree with this fix though :+1:

Reply:
I moved the change to another place, and also added a test.